azure-devops/create-vmss.ps1

```diff
@@ -304,6 +304,14 @@ Write-Progress `
 
 Restart-AzVM -ResourceGroupName $ResourceGroupName -Name $ProtoVMName
 
+####################################################################################################
+Write-Progress `
+  -Activity $ProgressActivity `
+  -Status 'Sleeping after restart' `
+  -PercentComplete (100 / $TotalProgress * $CurrentProgress++)
+
+Start-Sleep -Seconds 60

```

Comment:
A comment about why you added this would be good, however. (not going to block this PR on it, but it'd be nice)

Reply:
oh never mind, the progress message is sufficient.

Reply:
I can add a comment in a followup PR too. Thanks for approving the current PR.